pavelib/assets.py

```diff
@@ -80,12 +98,46 @@ def applicable_sass_directories(systems=None):
     if not systems:
         systems = ALL_SYSTEMS
     applicable_directories = []
+
     applicable_directories.extend(COMMON_SASS_DIRECTORIES)
     if "lms" in systems:
         applicable_directories.extend(LMS_SASS_DIRECTORIES)
     if "studio" in systems or "cms" in systems:
         applicable_directories.extend(CMS_SASS_DIRECTORIES)
+
+    # Put theme sass at the bottom so css files generated by theme sass overrides existing css styles.
     applicable_directories.extend(THEME_SASS_DIRECTORIES)
+
+    return applicable_directories
+
+
+def sass_lookup_directories(systems=None):
+    """
+    Determine the sass directories to be added to sass lookup paths.
+
+    Args:
+        systems: A list of systems (defaults to all)
+
+    Returns:
+        A list of SASS directories to be compiled.
+    """
+    if not systems:
+        systems = ALL_SYSTEMS
+    applicable_directories = []
+    applicable_directories.extend(COMMON_SASS_LOOKUP_DIRECTORIES)
+
+    # Put theme sass at the top so that theme directories have highest priority on sass file lookup.
+    applicable_directories.extend([sass_dir for sass_dir, _ in THEME_SASS_DIRECTORIES])
```

Comment:
I expected the logic to only add the LMS theme directory when "lms" was being compiled, and the same for CMS. Is it necessary to add all the themes all the time? Can you maintain ``LMS_THEME_SASS_DIRECTORIES`` and for CMS too?